SignalShareExtension/ShareViewController.swift

```diff
@@ -20,6 +20,15 @@ public class ShareViewController: UINavigationController, SAELoadViewDelegate, S
     override open func loadView() {
         super.loadView()
 
+        // We can't show the conversation picker until the DB is set up.
+        // Normally this will only take a moment, so rather than flickering and then hiding the loading screen
+        // We start as invisible, and only fade it in if it's going to take a while
+        self.view.isHidden = true
```

Comment:
@michaelkirk This is kind of weird but some of my comments (from an hour? ago) on this PR got lost.  I'm 99% sure I made them the right way.  Anyway, I'll try to recreate them now.

Reply:
Why use `isHidden` to hide?  Isn't that redundant with setting the alpha to zero?  Also, won't it interfere with the alpha animation below?

Reply:
You're right, it's not necessary. I've remove it. PTAL @charlesmchen